atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -1773,6 +1774,27 @@ private void alterGcAndTombstoneWithLock(
         }
     }
 
+    public void cleanUpSchemaMutationLockTablesState() throws Exception {
+        // get a unique table and create it if it doesn't exist
+        Set<TableReference> tables = lockTables.getAllLockTables();
+        TableReference unique;
+        if (tables.size() == 0) {
+            unique = lockTables.createLockTable();
+            log.info("There were no locks tables, so one was succesfully created: {}", unique.toString());
+            return;
+        }
+        unique = tables.iterator().next();
+        tables.remove(unique);
```

Comment:
I actually think we can remove _all_ the lock tables, including the "unique" one.  A new one will just get created on restart. There's no reason to create one here. 

That way, we can replace everything here with `dropTablesWithLock(lockTables.getAllLockTables())`